frontend/layouts/default.vue

```diff
@@ -194,7 +198,6 @@ export default {
     logout () {
       this.$store.commit('auth/logout')
       this.$notification.success(this.$i18n.t('auth.logout.success'))
-      this.$cookies.remove('oauth')
```

Comment:
why remove this line?

Reply:
its from a older version so...its pushing this change